hoodie-common/src/main/java/com/uber/hoodie/common/HoodieCleanStat.java

```diff
@@ -81,6 +82,46 @@ public String getEarliestCommitToRetain() {
         return new Builder();
     }
 
+    public HoodieCleanStat addDeletePathPattern(String deletePathPattern) {
```

Comment:
Corrected this. Using a private class called to gather info required to ultimately build a HoodieCleanStat.